services/airflow/dags/datenspende_vitaldata/data_update/download.py

```diff
@@ -0,0 +1,70 @@
+import polars
+import polars as po
+import glob
+from polars.datatypes import Utf8, Int64
+import ramda as R
+from typing import List, Tuple
+
+from dags.helpers.dag_helpers import download_7zfile, unzip_7zfile
+
+
+DataList = List[Tuple[str, List, polars.DataFrame]]
+
+
+@R.curry
+def download(access_config: dict, url: str) -> DataList:
+    return R.pipe(
+        download_7zfile(access_config),
+        unzip_7zfile(access_config),
+        load_files,
+        map_dict_to_list,
+    )(url)
```

Comment:
```suggestion
def download(access_config: dict, file_loader, url: str) -> DataList:
    return R.pipe(
        download_7zfile(access_config),
        unzip_7zfile(access_config),
        file_loader,
        map_dict_to_list,
    )(url)
```

Reply:
inject the file loader, so the download function can be used for both, the survey and the vital data updates. Then move the download function to the same place as the download7zfile and unzip_7zfile functions.